diff --color -Npur etcd-3.5.0-old/api/etcdserverpb/rpc.proto etcd-3.5.0/api/etcdserverpb/rpc.proto
--- etcd-3.5.0-old/api/etcdserverpb/rpc.proto	2021-06-16 05:34:52.000000000 +0800
+++ etcd-3.5.0/api/etcdserverpb/rpc.proto	2022-08-01 16:10:49.559459777 +0800
@@ -2,8 +2,8 @@ syntax = "proto3";
 package etcdserverpb;

 import "gogoproto/gogo.proto";
-import "etcd/api/mvccpb/kv.proto";
-import "etcd/api/authpb/auth.proto";
+import "kv.proto";
+import "auth.proto";

 // for grpc-gateway
 import "google/api/annotations.proto";
diff --color -Npur etcd-3.5.0-old/server/etcdserver/api/v3election/v3electionpb/v3election.proto etcd-3.5.0/server/etcdserver/api/v3election/v3electionpb/v3election.proto
--- etcd-3.5.0-old/server/etcdserver/api/v3election/v3electionpb/v3election.proto	2021-06-16 05:34:52.000000000 +0800
+++ etcd-3.5.0/server/etcdserver/api/v3election/v3electionpb/v3election.proto	2022-08-01 16:11:57.767456176 +0800
@@ -2,8 +2,8 @@ syntax = "proto3";
 package v3electionpb;

 import "gogoproto/gogo.proto";
-import "etcd/api/etcdserverpb/rpc.proto";
-import "etcd/api/mvccpb/kv.proto";
+import "rpc.proto";
+import "kv.proto";

 // for grpc-gateway
 import "google/api/annotations.proto";
diff --color -Npur etcd-3.5.0-old/server/etcdserver/api/v3lock/v3lockpb/v3lock.proto etcd-3.5.0/server/etcdserver/api/v3lock/v3lockpb/v3lock.proto
--- etcd-3.5.0-old/server/etcdserver/api/v3lock/v3lockpb/v3lock.proto	2021-06-16 05:34:52.000000000 +0800
+++ etcd-3.5.0/server/etcdserver/api/v3lock/v3lockpb/v3lock.proto	2022-08-01 16:12:29.867454481 +0800
@@ -2,7 +2,7 @@ syntax = "proto3";
 package v3lockpb;

 import "gogoproto/gogo.proto";
-import "etcd/api/etcdserverpb/rpc.proto";
+import "rpc.proto";

 // for grpc-gateway
 import "google/api/annotations.proto";
