--- a/build.sh
+++ b/build.sh
@@ -17,5 +17,5 @@
 # Set GO_LDFLAGS="-s" for building without symbols for debugging.
 # shellcheck disable=SC2206
 GO_LDFLAGS=(${GO_LDFLAGS:-} "-X=${VERSION_SYMBOL}=${GIT_SHA}")
-GO_BUILD_ENV=("CGO_ENABLED=0" "GO_BUILD_FLAGS=${GO_BUILD_FLAGS:-}" "GOOS=${GOOS}" "GOARCH=${GOARCH}")
+GO_BUILD_ENV=("CGO_ENABLED=1" "GO_BUILD_FLAGS=${GO_BUILD_FLAGS:-}" "CGO_CFLAGS=-D_FORTIFY_SOURCE=2 -O2 -fstack-protector-strong -fPIE" "GOOS=${GOOS}" "GOARCH=${GOARCH}")

 # enable/disable failpoints